spec/text/text_score/engine_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+require 'ronin/support/encoding/base64'
+
+describe Ronin::Support::Text::Scoring do
+  describe ".english_n_glyph_metric" do
+    context "when providing english text" do
+      it "must return a positive numeric score" do
+        expect(subject.english_n_glyph_metric("eeeeeeeeeee")).to be_a_kind_of(Numeric)
+        expect(subject.english_n_glyph_metric("eeeeeeeeeee")).to be > 0
+      end
```

Comment:
I would add more tests for positive-matching edge-cases such as:

* a single word
* two words
* a sentence
* a sentence with a comma
* a sentence with a semi-colon.
* two sentences.
* two sentences joined by a newline.